### DISCUSSION TITLE
Introducing Qiskit Office Hours!

### DISCUSSION THREAD

**Discussion by javabster** [Qiskit/.github#17] [Announcements]
Hello Qiskitters!

To help those of you who are active in Qiskit GitHub this year we are starting a pilot program of regular office hours with core maintainers. Twice a month a core maintainer will be hosting a 1hr webex session where anyone is free to drop in if they need support on a Qiskit GitHub issue. If you’re stuck, have a question or need help with a review come along and we’ll be happy to help you!

🕐  Timings:
2nd Monday of every month 2pm GMT / 9am EST
4th Monday of every month 5pm GMT / 12pm EST

📍 Location:
https://ibm.webex.com/ibm/j.php?MTID=mf3e20e95030999a4548ca6dec034fe42

The first session will be held Mon 14th Feb 2pm GMT / 9am EST - save it in your calendar! We’ll also send out slack reminders at the time of the session as well

**Comment by jwoehr** [comment]
Perhaps we could use an `#ibmq-faq` topic.
Locked for new responses except by certain individuals, but containing answers to "How many circuits can I run simulatenously on IBMQ Experience" etc.
Then those of us who answer those questions over and over again could just paste a link to the appropriate faq response.

**Reply by 1ucian0** [reply]
More than 3-year after, [here](https://github.com/orgs/Qiskit/discussions/categories/faq)!  :)

**Reply by jwoehr** [reply]
That seems just to point back to my same old topic, @1ucian0 , which is sadly out of date.

**Comment by Sumitchongder** [comment]
Hello Qiskit team,

My name is Sumit Chongder and my email is sumitchongder960@gmail.com. I have already submitted the Slack access form via qisk.it/join-slack, but I haven’t received an invitation yet even after a few days. Could you please check the status of my request or let me know if I need to take any further steps?

Thank you very much for your support!
